@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -140,11 +140,21 @@ export interface OmvDataSourceParameters {
 
     /**
      * Identifier used to choose OmvFeatureModifier, if undefined [[OmvGenericFeatureModifier]] is
-     * used. This parameter gets applied to the decoder used in the [[OmvDataSource]] which might
+     * used only if [[politicalView]] is undefined, otherwise [[OmvPointOfViewFeatureModifier]] will
+     * be used.
+     * This parameter gets applied to the decoder used in the [[OmvDataSource]] which might
```

Comment:
The issue here is that implementing  POV as a feature modifier effectively removes the ability of a user to add his own feature modifier, or forces him to make it a subclass of the POV modifier. I do not think that this is a wise decision.